lib/components/notifications-component.js

```diff
@@ -16,8 +16,13 @@ export default class NotificationsComponent extends BaseContainer {
 		return text;
 	}
 	selectCommentByText( commentText ) {
-		const commentSelector = by.xpath( `//div[normalize-space(text())='${commentText}']` );
-		return driverHelper.clickWhenClickable( this.driver, commentSelector );
+		const self = this;
+		const commentSelector = by.css( 'div > div.wpnc__text-summary > div:nth-child(1)' );
+		this.driver.findElement( by.css( 'div.wpnc__excerpt' ) ).getText().then( function( selectedCommentText ) {
+			if( selectedCommentText === commentText ) {
```

Comment:
Thanks for pointing that out @hoverduck! I've updated the code to use filter instead. It's my first time using it, so let me know if there's any issues with the implementation of it. Running the wp-notifications-spec locally, at least, passed.